### ISSUE TITLE
Adding filter option for products

### ISSUE THREAD

**Issue by MandeyMan** [JiyaGupta-cs/ShopNex#21]
Adding a search bar and a sort by option will increase the flexibility of this website 

please allow me this issue under JWOC 2024

**Comment by JiyaGupta-cs** [comment]
An issue is already opened for Sort feature
You can go ahead with the Filter option @MandeyMan

**Comment by JavidSumra** [comment]
Hey @JiyaGupta-cs Mam I Would Like to Work on This Issue Under JWOC 2024

**Comment by JiyaGupta-cs** [comment]
@JavidSumra Go ahead for search and filter option
@MandeyMan Unassigning you due to inactivity

**Comment by JiyaGupta-cs** [comment]
@JavidSumra Any update?

**Comment by JavidSumra** [comment]
Yeah @JiyaGupta-cs Started working on it but need some clarification on the search feature??

**Comment by JiyaGupta-cs** [comment]
@JavidSumra
Make an attribute of color and add to the product
And make a search option to select the colour
And likewise with the category and tags

**Comment by JiyaGupta-cs** [comment]
@JavidSumra 
![image](https://github.com/JiyaGupta-cs/ShopNex/assets/140608790/fdc0f807-4638-4873-9de5-ad78785129ec)

**Comment by JavidSumra** [comment]
Hey @JiyaGupta-cs Mam All the Data available is static i think so how can do this things?

**Comment by JiyaGupta-cs** [comment]
Just like quantity and other things
Make same for those also

**Comment by JiyaGupta-cs** [comment]
@JavidSumra Any Update?

**Comment by JavidSumra** [comment]
Hey @JiyaGupta-cs Mam due to some busy schedule I'll do it soon
